DbAccessor/src/Shared/RequestObjects.cs

```diff
@@ -36,65 +36,84 @@ public class UserRegistrationInfo
 
     public class PostDetails
     {
+        [JsonRequired]
         [JsonPropertyName("postId")]
-        public string PostId { get; set; }
+        public required Guid PostId { get; set; }
```

Comment:
The code was breaking and blocked here due to Deserialization of JSON while creating post:
![image](https://github.com/user-attachments/assets/204b3467-2db3-45f2-9f0f-70bdc2c202b9)
![image](https://github.com/user-attachments/assets/0f6674b7-bfd5-4cc7-bd25-8a38f6b4d817)

Since the postId is required and non-nullable, the deserialization always failed with the error above. The fix is, we have to change `Guid PostId` to `Guid? PostId` making it nullable which fixed the issue for me (tested locally end to end)

Reply:
Not sure how I missed that one, thanks for catching that